### PR TITLE
KULRICE-14145 : Maven Upgrade in support of Java 8

### PR DESCRIPTION
**Pull request by cwade179**

### CODE REVIEW

**Comment by cniesen** [comment]
Let's hold off on this merger until the ci integration tests pass.  Especially so we know if issues happened because of the Spring 4 upgrade or the Java 8 upgrade.

**Comment by cniesen** [comment]
ok to test

**Comment by cniesen** [comment]
As discussed during a standup, it's OK to upgrade eclipselink to a milestone version.  Please add a jira to upgrade it to the release version before we release 2.6.0.

Also it was mentioned that groovy doesn't need to be updated.  If so can you revert it back to a stable release.

File: pom.xml [hunk 1]
```diff
@@ -125,13 +131,13 @@
...
     <dwr.version>3.0.RC2</dwr.version>
-    <eclipselink.version>2.5.2</eclipselink.version>
+    <eclipselink.version>2.6.0-M3</eclipselink.version>
     <ehcache.version>2.6.9</ehcache.version>
     <filemonitor.version>1.0-kuali</filemonitor.version>
     <freemarker.version>2.3.20-patch2</freemarker.version>
     <lesscss.version>1.3.3</lesscss.version>
     <glassfish-javax-json.version>1.0.4</glassfish-javax-json.version>
-    <groovy.version>1.8.9</groovy.version>
+    <groovy.version>2.4.0-rc-2</groovy.version>
```
Comment (cniesen) [inline pom.xml]: Revert Groovy upgrade

**Comment by cwade179** [comment]
Do not merge this one, merge pull #51 instead

**Pull request closed without merging**